docs/getting_started/about_composer.md

```diff
@@ -43,26 +43,37 @@ By doing it this way you will need to execute further Composer commands using `p
     -   as Username – your Installation key found above on the *"Maintenance and Support agreement details"* page in the service portal
     -   as Password – the token password you retrieved in step 3.
 
-    #### Support agreement expiry
+    #### Note: Support agreement expiry
 
-    If your Support agreement expires, your authentication token(s) will no longer work. They will become active again if the agreement is renewed, but this process may take up to 24 hours. (If the agreement is renewed before the expiry date, there will be no disruption of service.)
+    If your Support agreement expires, your authentication token(s) will no longer work. They will become active again if the agreement is renewed, but this process may take up to 24 hours. _(If the agreement is renewed before the expiry date, there will be no disruption of service.)_
```

Comment:
@MagdalenaZuba how should the underscores be rendered (at the end)? On preview they seem not to be rendered.